src/odb/include/odb/db.h

```diff
@@ -7135,6 +7136,38 @@ class dbAccessPoint : public dbObject
   // User Code End dbAccessPoint
 };
 
+class dbBusPort : public dbObject
+{
+ public:
+  int getFrom() const;
+
+  int getTo() const;
+
+  dbModBTerm* getPort() const;
+
+  dbModBTerm* getMembers() const;
+
+  dbModule* getParent() const;
+
+  // User Code Begin dbBusPort
+  // get element by bit index in bus (allows for up/down)
+  // linear access
+  dbModBTerm* getBusIndexedElement(int index) const;
+
+  void setFirstMember(dbModBTerm*);
+  dbModBTerm* getFirstMember();
+  int getSize() const;
+  bool getUpdown() const;
+  // reallocate the bus port so all members sequential.
+  void Realloc();
```

Comment:
I know this is for future work but this should not be public even when implemented.

Reply:
removed